### ISSUE TITLE
How to set `logoUrl` to show the logo image?

### ISSUE THREAD

**Issue by anikfal** [Unidata/tds#483]
According to https://docs.unidata.ucar.edu/tds/5.2/userguide/tds_config_ref.html#server-information:

> The best way to use your own logo is to put it in the ${tds.content.root.path}/thredds/public/ directory, and specify it in serverInformation as /thredds/<name>, e.g.:
> <logoUrl>/thredds/yourIcon.gif</logoUrl>

I tried both PNG and GIF images. However, the image of the logo could not be shown by this configuration.

**Comment by haileyajohnson** [comment]
It seems those instructions are not correct, the right place to put your image is under your tomcat `webapps/thredds` directory. I've just pushed changes to our documentation to correct that. Thanks!

**Comment by anikfal** [comment]
I think `thredds-5.4.war` has problems regarding this issue. Even after following the new configurations you recommended, the logo could not be displayed. However, after deploying `thredds-5.5-SNAPSHOT.war`, the logo appeared.

With `thredds-5.4.war`, images can be served by URL (e.g. `my_ip_address/thredds/logo_image.png`). But it seems, they can not be displayed on the web page.

For your information, I deployed `thredds-5.5-SNAPSHOT.war` as `thredds.war`. But for `thredds-5.4.war`, I deployed it without any change.
